projects/plugins/crm/includes/ZeroBSCRM.Database.php

```diff
@@ -879,6 +880,31 @@ function zeroBSCRM_createTables(){
   COLLATE = ".$collation.";"; 
   zeroBSCRM_db_runDelta($sql);
 
+	// Add table to store automation workflows.
+	// phpcs:disable WordPress.NamingConventions.ValidVariableName.VariableNotSnakeCase
+	$sql = 'CREATE TABLE IF NOT EXISTS ' . $ZBSCRM_t['automation-workflows'] . '(
+	`id` INT NOT NULL AUTO_INCREMENT,
+	`zbs_site` INT NOT NULL,
+	`zbs_owner` INT NOT NULL,
+	`name` VARCHAR(255) NOT NULL,
+	`description` VARCHAR(255) NULL DEFAULT NULL,
+	`category` VARCHAR(255) NOT NULL,
+	`triggers` LONGTEXT NOT NULL,
+	`initial_step` LONGTEXT NOT NULL,
+	`active` TINYINT(1) NOT NULL DEFAULT 0,
+	`version` INT(14) NOT NULL DEFAULT 1,
+	`created_at` INT(14) NOT NULL DEFAULT CURRENT_TIMESTAMP,
+	`updated_at` INT(14) NOT NULL DEFAULT CURRENT_TIMESTAMP,
```

Comment:
I wonder if this should be similar to the following?

```suggestion
	`created_at` TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP,
	`updated_at` TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP,
```

Reply:
This^^. For using `CURRENT_TIMESTAMP` the column type should be `TIMESTAMP` OR `DATETIME`.

Also, for the `updated_at` column, after the `DEFAULT CURRENT_TIMESTAMP` clause we could add this `ON UPDATE CURRENT_TIMESTAMP`

Reply:
Also this rule:

`Incorrect table definition; there can be only one TIMESTAMP column with CURRENT_TIMESTAMP in DEFAULT or ON UPDATE clause`

Reply:
Semantically, I'd prefer to leave `updated_at` as `NULL` until it's been updated (vs. just created).

Reply:
Looks like it is now set to 0.